google-cloud-bigtable/lib/google/cloud/bigtable/table.rb

```diff
@@ -67,7 +75,7 @@ class Table
         def initialize grpc, service, view: nil
           @grpc = grpc
           @service = service
-          @view = view || :SCHEMA_VIEW
+          @loaded_views = Set[view || :SCHEMA_VIEW]
```

Comment:
How are we confident that the contents of `@grpc` and `@loaded_views` (i.e. the passed in `view`) are in sync? Could we have a case where someone calls `Table.from_grpc` and passes in a `grpc` object that has only the `NAME_ONLY` fields populated, but doesn't pass in the `view:` argument, so `@loaded_views` ends up thinking that the default of `SCHEMA_VIEW` is already populated even though it isn't?

Reply:
I like it!